app/assets/stylesheets/components/_alert.scss

```diff
@@ -23,3 +23,9 @@
     transform: translateY(-50%);
   }
 }
+
+.usa-alert__body {
+  ul {
+    margin-bottom: 0px;
```

Comment:
We do. Let me play around and see if one can work. I think I'd have to add that class into the translation files though, and I'm not sure I want to do make that content even messier.

Reply:
> [This comment](https://github.com/18F/identity-idp/blob/5fa807f46c021e7241baa2e2f56065f8d354de0a/app/assets/stylesheets/email.css.scss#L159-L160) suggests the full design system might not be imported so that might be necessary (unless it's an outdated comment).

That stylesheet is specific for email messages.